recipes/fractal-client/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "fractal-client" %}
+{% set version = "1.3.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fractal_client-{{ version }}.tar.gz
+  sha256: 7af2aafda5aff223a622ad058d240d0a3718d13cf2599d332c7e02cb36f2142e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - fractal = fractal_client.__main__:run
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.9.0,<4.0.0
+    - python-dotenv >=0.20.0,<0.21.0
+    - rich >=12.5.1,<13.0.0
+    - httpx >=0.23.0,<0.24.0
+    - pyjwt >=2.4.0,<3.0.0
+    - packaging >=23.1.0,<24.0.0
+
+test:
+  imports:
+    - fractal_client
+  commands:
```

Comment:
Can you put in a check of the cli under the `commands` test. Something like `- fractal --help` or similar.

Reply:
Of course, thanks.
Fixed with https://github.com/conda-forge/staged-recipes/pull/24101/commits/fb09f00875a0d5a63d36fdc7ecbce9bd8eded479.